Yoast/javascript
# P1 34 change structure from @yoast/components package

## Summary
Change the structure of the release `@yoast/components` package. This makes sure that the linked monorepo and the released package have the same structure.

⚠️ ⚠️ ⚠️ needs a major version release for @yoast/components ⚠️ ⚠️ ⚠️ 


This PR can be summarized in the following changelog entry:
* [@yoast/components] ⚠️ ⚠️ ⚠️ needs a major version release
* [@yoast/components] Change the structure of the released packages. Everything is now included in a `src` directory. 

## Test instructions

This PR can be tested by following these steps:
Not sure how this can be tested other than actually creating a new package.

* Go to the `packages/components` directory.
* Run `yarn prepublishOnly`
* Verify this works and generated a `dist` folder.
* Verify that the dist folder holds `.babelrc` and `package.json` files and a `src` directory.
* Verify that the `package.json`'s `main` entry points to `src/index.js`.

## Quality assurance
